### DISCUSSION TITLE
Translating Markdown content

### DISCUSSION THREAD

**Discussion by rtsisyk** [WeblateOrg/weblate#8424] [Q&A] [answered]
I would like to use Weblate to translate Markdown files for Zola/Huga/Jekyll static web-site generators. I have the following structure:

- index.md - English
- index.de.md - German
- index.tr.md - Turkish
- ...
- page1.md - English
- page1.de.md - German
- page1.tr.md - Turkish
- ...

Is Weblate a good tool for this task?

**Comment by nijel** [comment, marked as answer]
Currently, the recommended workflow for this is using po4a.

Native Markdown support might eventually land in Weblate, but nobody is working on that. See https://github.com/WeblateOrg/weblate/issues/3106

**Reply by rtsisyk** [reply]
> Currently, the recommended workflow for this is using po4a.

Thanks for the idea! It [works like a charm](https://hosted.weblate.org/projects/organicmaps/website/). Feedback [is very positive]( https://github.com/organicmaps/organicmaps/discussions/4076)!
